scripts/tests/ota_test.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
 PASSCODE=${1:-20202021}
-DISCRIMINATOR=${2:-42}
-UDP_PORT=${3:-5560}
+REQUESTOR_DISCRIMINATOR=${2:-42}
+REQUESTOR_UDP_PORT=${3:-5540}
```

Comment:
I'm a little concerned that requestor is always resolving to 5540. Why is there a difference from provider such that provider can use a non-default port?

Reply:
I don't know why there is a difference, but that is exactly what I'm seeing. 


Using $PROVIDER_UDP_PORT = 5540
[1649309677.671335][41628:41636] CHIP:TOO: Discovered Device: fe80::8840:85ff:fe05:7d5a:5540
Comissioning PASS

Using $REQUESTOR_UDP_PORT = 5560 
[1649309678.351785][41645:41653] CHIP:TOO: Discovered Device: fe80::54b1:4aff:fe40:40d:5540
Commissioning FAILS




Using $PROVIDER_UDP_PORT = 5565 
[1649351723.806011][41626:41634] CHIP:TOO: Discovered Device: fe80::f45e:66ff:fe3b:da3b:5565
Commissioning PASS

Using $REQUESTOR_UDP_PORT = 5540
[1649351724.496417][41643:41651] CHIP:TOO: Discovered Device: fe80::f4d5:40ff:fe37:735b:5540
Commissioning PASS

Reply:
I'm worried this is hiding an underlying issue.

Reply:
What are the steps to reproduce the failure?  I applied this PR, set `REQUESTOR_UDP_PORT=5560` and `PROVIDER_UDP_PORT=5565` in the script, ran `bash scripts/tests/ota_test.sh`, it said "Test passed".

@mykrupp

Reply:
I only see this in the Linux CI workflow. I can run and pass as you stated on my mac as well.

Reply:
Could this be mDNS isolation issues on linux as well?

Linux integration tests use net namespaces to isolate networks to ensure no crosstalk, maybe we should do the same for OTA.